lib/AST/NameLookup.cpp

```diff
@@ -1499,9 +1499,13 @@ bool DeclContext::lookupQualified(Type type,
     if (tracker)
       tracker->addUsedMember({current, member.getBaseName()},isLookupCascading);
 
-    // Make sure we've resolved implicit constructors, if we need them.
-    if (member.getBaseName() == ctx.Id_init && typeResolver)
-      typeResolver->resolveImplicitConstructors(current);
+    // Make sure we've resolved implicit members, if we need them.
+    if (typeResolver) {
+      if (member.getBaseName() == ctx.Id_init)
+        typeResolver->resolveImplicitConstructors(current);
+
+      typeResolver->resolveImplicitMember(current, member);
```

Comment:
Well, you're synthesizing members, right? So what happens when you ask for `init(from:)`from outside the type and it goes to look to see if you have a `CodingKeys`, and then that ends up checking to see if you have an `init(from:)`?

I mean, it's clearly *not* doing that, but it would be good to know why.